packages/luna/src/locales/ar-eg.json

```
@@ -340,6 +351,8 @@
   "labels.update": "تعديل",
   "labels.new.password": "كلمة السر الجديدة",
   "labels.repeat.new.password": "أعد كتابة كلمة السر الجديدة",
+  "global.createdAt": "تاريخ الإنشاء",
+  "global.updatedAt": "تاريخ التعديل",
```

Let's use the dash-named notation when naming message ids. 

```suggestion
  "global.created-at": "تاريخ الإنشاء",
  "global.updated-at": "تاريخ التعديل",
```